deployment/frontend/src/components/team/CreateTeam.tsx

```diff
@@ -0,0 +1,95 @@
+import Container from "../_shared/Container";
+import { InputGroup } from "../_shared/InputGroup";
+import Select from "../_shared/SimpleSelect";
+import { Input } from "../_shared/SimpleInput";
+import UploadButton from "../datasets/sections/datafiles/Upload";
+import { Breadcrumbs } from "../_shared/Breadcrumbs";
+
+const links = [
+  { label: "Teams", url: "/teams", current: false },
+  { label: "Create a team", url: "/teams/new", current: true },
+];
+
+export default function CreateTeam() {
+  return (
+    <>
+      <Breadcrumbs links={links} />
+      <Container className="mb-20 font-acumin">
+        <h1 className="mb-[2rem] text-[1.57rem] font-semibold">
+          Create a team
+        </h1>
+        <div className="w-full border-b border-blue-800 shadow">
+          <div className="px-8">
+            <div className="grid grid-cols-1 items-start gap-x-24 gap-y-4 py-5 md:grid-cols-2">
+              <div className="flex flex-col justify-start gap-y-4">
+                <InputGroup label="Name" required>
+                  <Input
+                    name="name"
+                    maxWidth="100%"
+                    placeholder="Name"
+                    type="text"
+                  />
+                </InputGroup>
+                <InputGroup label="Title" required>
+                  <Input
+                    name="title"
+                    maxWidth="100%"
+                    placeholder="My dataset"
+                    type="text"
+                  />
+                </InputGroup>
+                <InputGroup label="URL" required>
+                  <Input
+                    name="url"
+                    maxWidth="100%"
+                    placeholder="name-of-dataset"
+                    type="text"
+                    className="!pl-[4.8rem]"
```

Comment:
I think this padding left needs to be smaller there is quite a big gap
![image](https://github.com/wri/wri-odp/assets/11317382/da7e5e7e-6146-493a-baea-6b6c1b0ad653)